set_environment.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
 export LICENSE_ID=
-export SPARCDIR=${VSC_DATA}/cryoSPARC
+export SPARCDIR=${VSC_DATA}/apps/${VSC_INSTITUTE_CLUSTER}/cryoSPARC
 export CREDIT_ACCOUNT=
-export SSD_PATH='${VSC_SCRATCH_NODE}'
+# prepare SSD setup
+cd ${VSC_SCRATCH_NODE}
+test -d ./ssddir || lfs setstripe -E -1 --pool=ssd -c 4 -S 1M ssddir
+export SSD_PATH='${VSC_SCRATCH_NODE}/ssddir'
```

Comment:
I don't think these changes are necessary, `$VSC_SCRATCH_NODE` is a local SSD disk both on KU Leuven clusters and on Hortense, so it suffices to point `SSD_PATH` to it.
The `lfs setstripe` command only applies to directories on Lustre, which is not the case for `$VSC_SCRATCH_NODE`.

Reply:
sure. I was not aware of that. The local node scratch is not properly represented on  vscdocs, so, i did not have the best reference. See b70071b.